enas/cifar10/data_utils.py

```diff
@@ -1,8 +1,9 @@
 import os
 import sys
-import cPickle as pickle
+import _pickle as pickle
```

Comment:
this is incompatible with python2, this needs to be updated to support both python2 and python3 (I know we discussed this, I'm just doing a code review and I'm including this for completeness)